MAUI/Barcode/two-dimensional-symbology.md

```diff
@@ -0,0 +1,273 @@
+                                ShowText="True"
+                                HeightRequest="300" 
+                                WidthRequest="300">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:QRCode CodeVersion="Version09"/>
+        </barcode:SfBarcodeGenerator.Symbology>
+     </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="7" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 300;
+    barcode.WidthRequest = 300;
+    barcode.Value = "www.syncfusion.com";
+    barcode.Symbology = new QRCode() 
+    { 
+        CodeVersion = QRCodeVersion.Version09 
+    };
+
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator QR Code Version09](images/two-dimensional-symbology/maui-qr-code-version09.png)
+
+## Data Matrix
+
+`Data Matrix` barcode is a two dimensional barcode that consists of a grid of dark and light dots or blocks forming a square or rectangular symbol. The data encoded in the barcode can either be numbers or alphanumeric. They are widely used in printed media such as labels and letters. You can read it easily with the help of a barcode reader and mobile phones.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="6" %}
+
+    <barcode:SfBarcodeGenerator Value="www.syncfusion.com"
+                                ShowText="True"
+                                HeightRequest="300" 
+                                WidthRequest="300">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:DataMatrix/>
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="5" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 300;
+    barcode.WidthRequest = 300;
+    barcode.Value = "www.syncfusion.com";
+    barcode.Symbology = new DataMatrix();
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Data Matrix Symbology](images/two-dimensional-symbology/maui-data-matrix.png)
+
+### Data matrix format
+
+Length - Data Matrix can store up to 2335 alphanumeric characters or 3116 numbers from the ASCII range.
+
+Type - Data Matrix supports the following data types,
+
+* Numeric
+* Alpha Numeric
+* Byte
+
+The `Size` depends upon the length and data type of provided input values.
+
+### Encoding Methods
+
+Data Matrix supports following `Encoding` types,
+
+* Auto
+* ASCII
+* ASCIINumeric
+* Base256
+
+By default, the encoding types is `Auto`.
+
+When the encoding type is `ASCII`, the code word will be calculated as like follows,
+
+Code word = ASCII value + 1.
+
+The `ASCII` value ranges from 0 to 127
+
+When the encoding type is `Base256`, then the first code word will be calculated with the value 235 and the second code value is calculated as ASCII value - 127.
```

Comment:
When the encoding type is `Base256`, then the first code word is calculated with the value 235, and the second code value is calculated with the ASCII value - 127.

Reply:
Changes updated.